spec-dtslint/observables/from-spec.ts

```diff
@@ -36,13 +36,21 @@ it('should accept an array of Observables', () => {
   const o = from([of(1), of(2), of(3)]); // $ExpectType Observable<Observable<number>>
 });
 
-it('should accept an array of Inputs', () => {
-  const iterable = (function*() {
-    yield 42;
-  }());
+// TODO(benlesh): enable this test, once the issue is resolved upstream
 
-  const o = from([of(1), ['test'], iterable]); // $ExpectType Observable<IterableIterator<number> | Observable<number> | string[]>
-});
+// NOTE: It appears to be working, it's just that dtslint sometimes says it wants
+// Observable<IterableIterator<number> | Observable<number> | string[]>
+// and if you switch it to that, it wants
+// Observable<Observable<number> | IterableIterator<number> | string[]>
+// and vica versa.
+
+// it('should accept an array of Inputs', () => {
+//   const iterable = (function*() {
+//     yield 42;
+//   }());
+
+//   const o = from([of(1), ['test'], iterable]); // $ExpectType Observable<IterableIterator<number> | Observable<number> | string[]>
```

Comment:
IIRC, dtslint will still see `$ExpectType` and will complain about not being able to find a missing node. You have to change it to `$_TODO_ExpectType` or something.

Reply:
If different TS versions report the union type in different orders, this is going to be really annoying/painful. There's a related dtslint PR (https://github.com/Microsoft/dtslint/pull/61) that was abandoned.